uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/leave-type-wizard.module.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-env amd */
+
+define([
+  'common/angular',
+  'leave-absences/leave-type-wizard/components/leave-type-wizard.component',
+  'common/directives/help-text.directive',
+  'leave-absences/leave-type-wizard/leave-type-wizard.core'
+], function (angular, LeaveTypeWizardComponent) {
+  angular.module('leave-type-wizard', [
+    'common.directives',
+    'common.services',
+    'leave-type-wizard.core'
+  ])
+    .component(LeaveTypeWizardComponent.__name, LeaveTypeWizardComponent);
```

Comment:
Just noticed this: with AngularJS 1.7 we don't need to use this custom `__name` property anymore. You can register the component by simply doing
```js
.component(LeaveTypeWizardComponent)
```
if `LeaveTypeWizardComponent` is an object where the key is the name of the component and the value is the component itself, please see [here](https://github.com/compucorp/civihr-tasks-assignments/blob/staging/uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-notification-badge/components/tasks-notification-badge.component.js#L28-L36) and [here](https://github.com/compucorp/civihr-tasks-assignments/blob/staging/uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-notification-badge/tasks-notification-badge.module.js#L15) for an example

Please note that this applies for anything in angular: services, directives, controllers, etc